cafe_information

```diff
@@ -0,0 +1,21 @@
+create table CafeInSanFrancisco (
+	name varchar(32) NOT NULL,
+	address varchar(64) NOT NULL,
```

Comment:
Same as above. Not too small?

Reply:
@harrydrippin I'll change into 512 characters.